pkg/url/url.go

```diff
@@ -100,6 +103,79 @@ func GetIngress(kClient *kclient.Client, envSpecificInfo *envinfo.EnvSpecificInf
 	return iextensionsv1.Ingress{}, errors.New(fmt.Sprintf("the url %v does not exist", urlName))
 }
 
+// GetContainer returns Docker URL definition for given URL name
+func GetContainerURL(envSpecificInfo *envinfo.EnvSpecificInfo, urlName string, componentName string) (URL, error) {
```

Comment:
please write a unit test for this function by passing a fake client into the function

Reply:
Added unit tests for these two functions.